released/packages/coq-vst-zlist/coq-vst-zlist.2.13/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "A list library indexed by Z type, with a powerful automatic solver"
+authors: [
+  "Qinshi Wang"
+  "Andrew W. Appel"
+]
+maintainer: "VST team"
+homepage: "http://vst.cs.princeton.edu/"
+dev-repo: "git+https://github.com/PrincetonUniversity/VST.git"
+bug-reports: "https://github.com/PrincetonUniversity/VST/issues"
+license: "BSD-2-Clause"
+
+build: [
+  [make "-C" "zlist" "-j%{jobs}%"]
+]
+run-test: []
+install: [make "-C" "zlist" "install"]
+depends: [
+  "coq" {>= "8.16.1"}
```

Comment:
```suggestion
  "coq" {>= "8.16.1" & < "8.19~"}
```
Maybe good to use the same Coq bounds for all packages here? We can hopefully loosen the bounds for the next Coq version for all these packages all at once.